pkg/logs/config/config.go

```diff
@@ -73,9 +82,14 @@ func BuildEndpoints() (*client.Endpoints, error) {
 		main.Port = 443
 		useSSL = true
 	default:
-		// datadog settings
-		main.Host = coreConfig.Datadog.GetString("logs_config.dd_url")
-		main.Port = coreConfig.Datadog.GetInt("logs_config.dd_port")
+		// If no proxy is set, we default to 'logs_config.dd_url' if set, or to 'site'.
+		// if none of them is set, we default to the US agent endpoint.
+		main.Host = coreConfig.GetMainEndpoint(endpointPrefix, "logs_config.dd_url")
+		if port, found := logsEndpoints[main.Host]; found {
+			main.Port = port
+		} else {
+			main.Port = coreConfig.Datadog.GetInt("logs_config.dd_port")
```

Comment:
what happens if `logs_config.dd_port` is not set ?

Reply:
It has a default value: https://github.com/DataDog/datadog-agent/pull/2824/files#diff-091ace02b49a46f55f2586cfec89a160R308